migration-guide/now-to-next.adoc

```diff
@@ -0,0 +1,237 @@
+
+. Create the project using Quarkus with the Maven commands below, this will create a Quarkus project called `my-sample-decision-service`, which will be versioned `1.0.0-SNAPSHOT`, including the extensions `kogito-quarkus, dmn, resteasy-reactive-jackson, quarkus-smallrye-openapi, quarkus-smallrye-health`.  This will create a Quarkus DMN project with the OpenAPI components to get the OpenAPI end points easily with health checks when deploying to Kubernetes:
+
++
+[source,console,subs="+attributes"]
+----
+mvn io.quarkus:quarkus-maven-plugin:2.16.7.Final:create \
+    -DprojectGroupId=com.ibm.bamoe.samples \
+    -DprojectArtifactId=my-sample-decision-service \
+    -DprojectVersion=1.0.0-SNAPSHOT \
+    -DplatformVersion=2.16.7.Final \
+    -Dextensions=kogito-quarkus,dmn,resteasy-reactive-jackson,quarkus-smallrye-openapi,quarkus-smallrye-health
+----
+
+[NOTE]
+====
+As you can see from the Maven command above, there are a number of possible `extensions` that can be added to the project in order to add some sort of specific functionality.  You can find more information on available extensions at the following locations:  https://quarkus.io/extensions/[Quarkus Extensions] and https://quarkus.io/extensions/?search-regex=kogito[Kogito Extensions].
+====
+
+When you create this project you should get a bunch of Maven artifacts start to stream in your console that are being pulled and ultimately are left with a console message like below:
+
+[source,console]
+----
+[INFO]
+[INFO] ========================================================================================
+[INFO] Your new application has been created in /Users/developer/my-sample-decision-service
+[INFO] Navigate into this directory and launch your application with mvn quarkus:dev
+[INFO] Your application will be accessible on http://localhost:8080
+[INFO] ========================================================================================
+[INFO]
+[INFO] ------------------------------------------------------------------------
+[INFO] BUILD SUCCESS
+[INFO] ------------------------------------------------------------------------
+[INFO] Total time:  24.548 s
+[INFO] Finished at: 2022-09-27T10:22:31-04:00
+[INFO] ------------------------------------------------------------------------
+----
+
+[start=3]
+.  Add the following `properties` to the `pom.xml` file, using the row numbers below as a guide.  This specifiies the Maven coordinates for the {PRODUCT_SHORT} Maven Bill-of-Materials (BOM), which manages all of the associated depenency versions to a specific {PRODUCT_SHORT} release:
+
+[source,xml]
+----
+17        <surefire-plugin.version>3.0.0-M7</surefire-plugin.version>
+18  +++   <kogito.bom.group-id>com.ibm.bamoe</kogito.bom.group-id>
+19  +++   <kogito.bom.artifact-id>bamoe-bom</kogito.bom.artifact-id>
+20  +++   <kogito.bom.version>9.0.0.Final</kogito.bom.version>
```

Comment:
Shouldn't be this `9.0.0.Final` be declared as some variable? Like `VERSION`, `BOM_VERSION`, `KOGITO_BOM_VERSION` ... so we can easily keep docs updated? I mean from ascidoc point of view.

Reply:
We do not currently have it in `partials/attributes.adoc`, but it 100% should be in there. Good catch, @jomarko

Reply:
This is in particls/attributes.adoc as BOM_VERSION, I have updated the table to use the variable.